### ISSUE TITLE
Performance of llava-med

### ISSUE THREAD

**Issue by ymliu529** [DCDmllm/HealthGPT#23]
Why is the performance of Llava med in Table 1 much lower than that in the original paper?

**Comment by Lin-Tianwei** [comment]
Llava-med is optimized based on the corresponding benchmark training set, while HealthGPT focuses more on generalization.

**Comment by ymliu529** [comment]
thanks for your response. it that mean the results of llava-med in table 1 are reported by reproducing llava-med but not fine-tuning on VQA-RAD or SLAKE?

**Comment by Lin-Tianwei** [comment]
Yes, continuous training on datasets like SLAKE can lead to structural degradation in the quality of generated content. We have not adopted this approach.

**Comment by ymliu529** [comment]
Another question is that, as observed, PubMedVision dataset has some multi image inputs. How does HealthGPT handle this?

**Comment by Lin-Tianwei** [comment]
Multi-image training introduces padding, which increases training time. HealthGPT does not use multi-image input, but the ability to handle multiple images can be easily extended by concatenating image features.
